meetalva/alva
# feat: edit property with unknown type

* All properties without successful detection are now deemed "unknown"
* Alva renders a "Code properties" detail element if such properties are found
* Input method are small auto-resizing Monaco instances

# Test

* Download attached file: [unknown-properties.alva.zip](https://github.com/meetalva/alva/files/2650494/unknown-properties.alva.zip)
* Open with deployed version: https://alva-issues-669.surge.sh
* Select button 
* See changed properties pane
